docs/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+- [ ] Link to pivotal/JIRA issue
+- [ ] What was done?
+    - Explain the implementation goals being solved or the feature with the reviewer in mind
+    - Mention any relevant issues or technical gotchas
```

Comment:
what is a technical gotcha?

Reply:
Things like: An assumption you initially that was changed after making this PR that would be good to share.  Would calling it 'insights' make it clearer?